websites/W/Wolfy/presence.ts

```diff
@@ -9,6 +9,11 @@ let path,
 	cp: number,
 	currTime: string;
 
+const waitingString: Record<string, string> = {
```

Comment:
```suggestion
const waitingString = {
```

Reply:
The program don't compile without this type specefication because it's used like an array with string keys like you can see line 210

```ts
presenceData.state === waitingString[document.querySelector("html").lang]
```

Reply:
then you have to check if the query selector is either "en" or "fr" for type safety, this is not the solution

Reply:
I did in my last commit 👍